src/blocks/OurTeam/index.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import styled from 'styled-components'
+import PreTitle from 'components/PreTitle'
+import Grid from 'components/Grid'
+import get from 'utils/get'
+import Typography from 'components/Typography'
+import shuffle from 'utils/shuffle'
+import TeamMemberList from './TeamMemberList'
```

Comment:
```suggestion
import React from 'react'
import styled from 'styled-components'
import PreTitle from 'components/PreTitle'
import Grid from 'components/Grid'
import Typography from 'components/Typography'
import shuffle from 'utils/shuffle'
import get from 'utils/get'
import TeamMemberList from './TeamMemberList'
```

Why is this a relative link `import TeamMemberList from './TeamMemberList'` ?

Reply:
I splitted the blocks that were too big in small files, and put them inside a folder of the block's name. Since these components are not gonna be used elsewhere, I chose to import them relatively :) Plus, this way, if I move the folder someday, the imports between each files of the same block won't be broken.